admin/app/components/certification-session-report.js

```diff
@@ -4,60 +4,47 @@ import _ from 'lodash';
 
 export default Component.extend({
 
-  // Element
   classNames: ['certification-session-report'],
 
-  // Props
-  candidates: null,
-
-  // CPs
-  incomplete: computed('candidates', function() {
-    return this.candidates.filter((candidate) => {
-      return !candidate.lastName || candidate.lastName.trim() === ''
-        || !candidate.firstName || candidate.firstName.trim() === ''
-        || !candidate.birthdate || candidate.birthdate.trim() === ''
-        || !candidate.birthplace || candidate.birthplace.trim() === ''
-        || !candidate.certificationId || candidate.certificationId.trim() === '';
+  outOfSessionCertifications: computed('sessionId,certifications,certifications.[]', function() {
```

Comment:
Pour info, suite à notre discussion, d'après https://guides.emberjs.com/v2.12.0/object-model/computed-properties-and-aggregate-data/:

> Computed properties dependent on an array using the [] key will only update if items are added to or removed from the array, or if the array property is set to a different array.

Donc si un élément de `certifications` est modifié, la fonction computed ne sera exécutée.

Reply:
Je trouve la syntaxe `computed('sessionId', 'certifications', 'certifications.[]' ...` plus explicite

Reply:
Ok je vais en parler avec la team voir ce qu'on préfère. Merci de tes retours.